Book_my_show_schema.txt

```diff
@@ -0,0 +1,86 @@
+create table Users(
+userid int primary key identity(1,1),
+Name varchar(50) not null,
+phone_number nvarchar(15),
+email varchar(254) unique,
+password varchar(50) not null);
+
+
+create table Movies(
+Movieid int primary key identity(1,1),
+Genre varchar(20) not null,
+Duration Time not null,
+language varchar(50),
```

Comment:
Normalization

Reply:
modified.
CREATE TABLE Movies (
Movieid INT PRIMARY KEY IDENTITY(1,1),
MovieName VARCHAR(50) NOT NULL,
Duration TIME NOT NULL,
ReleaseDate DATE NOT NULL
);
CREATE TABLE MovieLanguages (
    MovieID INT NOT NULL,
    LanguageID INT NOT NULL,
    PRIMARY KEY (MovieID, LanguageID),
    FOREIGN KEY (MovieID) REFERENCES Movies(Movieid),
    FOREIGN KEY (LanguageID) REFERENCES Languages(LanguageID)
);